File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.py
def solution(s):
    answer = ''
    tmp = s.split(" ")
    
    for word in tmp:
        for index, alphabet in enumerate(word):
            if index == 0:
                if alphabet.isnumeric() or alphabet == " ":
                    answer += alphabet
                else:
                    if alphabet.islower():
                        answer += alphabet.upper()
                    else:
                        answer += alphabet
            else:
                if alphabet.isupper():
                    answer += alphabet.lower()
                else:
                    answer+=alphabet
        answer += " "
    return answer[:-1]

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    var answer = '';
    var tmp = s.split(" ")
    for (let i=0; i<tmp.length;i++) {
        for (let j=0; j<tmp[i].length; j++) {
            if (j==0) {
                if (tmp[i][j] === tmp[i][j].toLowerCase()) {
                    answer = answer + tmp[i][j].toUpperCase()
                }
                else {
                    answer = answer + tmp[i][j]
                }
            }
            else {
                if (tmp[i][j] === tmp[i][j].toUpperCase()) {
                    answer = answer + tmp[i][j].toLowerCase()
                }
                else {
                    answer = answer + tmp[i][j]
                }
            }
        }
        answer = answer + " "
    }
    answer = answer.slice(0, answer.length - 1)
    return answer;
}